vuln-reach/Cargo.toml

```
@@ -2,6 +2,12 @@
 name = "vuln-reach"
 version = "0.1.0"
 edition = "2021"
+description = "Code reachability path analysis."
+authors = ["Phylum, Inc. <engineering@phylum.io>"]
+repository = "https://github.com/phylum-dev/vuln-reach"
+documentation = "https://docs.rs/vuln-reach"
+license-file = "../LICENSE"
+rust-version = "1.70"
```

Can we use something that is a bit older than latest?

```suggestion
rust-version = "1.68"
```

---

Sure!